### ISSUE TITLE
ancs service discovered but no notification received

### ISSUE THREAD

**Issue by xilin** [sandeepmistry/node-ancs#10]
I'm running the project in Raspbian with Raspberry 3.
I can see the output in console:

```
services and characteristics discovered
```

And I can print the characteristics in `discoverServicesAndCharacteristics` method.

But I never see the output of `onNotification` or `onData` method.

What maybe the reason I cannot see notification?

**Comment by sandeepmistry** [comment]
@xilin this module might be incompatible with newer versions of iOS.

Could you please provide an HCI dump capture: `sudo hcidump -t -x`

**Comment by sandeepmistry** [comment]
Closing this for now due to lack of activity.

**Comment by xilin** [comment]
Sorry for the late reply.
My Raspbian cannot work for some times. And today I got the chance and change to use OSMC as my system.
After pairing my iPhone using OSMC's build in bluetooth util, I can connect to iPhone using node-ancs and finally I can see the output of ANCS now!

Thanks for the great work.

**Comment by sandeepmistry** [comment]
@xilin thanks for the follow up.

Do you have some specific instructions you share on the steps taken to make it work?

**Comment by xilin** [comment]
@sandeepmistry The OSMC system is almost function ready when installed.

I did use `systemctl status bluetooth` to check the bluetooth status, and tried `sudo systemctl start bluetooth`/`sudo systemctl stop bluetooth` sometimes before node-ancs can work.

But finally I found out that I just need to go to the bluetooth setting page([Here is a instruction](https://osmc.tv/wiki/general/connecting-a-bluetooth-device/)) and pair my iPhone with Raspberry Pi. 

After the pairing, I run `sudo node test.js` and the notifications start to scroll on the screen immediately.
